shared-libs/transitions/src/transitions/registration.js

```diff
@@ -185,6 +186,27 @@ const setBirthDate = doc => {
   }
 };
 
+const getPhoneNumber = doc => {
+  if (!doc || !doc.fields) {
+    return '';
+  }
+  return doc.fields.phone_number;
+};
+
+const setPhoneNumber = options => {
+  const doc = options.doc;
+  const phoneNumber = getPhoneNumber(doc);
+  // By default for a valid phone number SmsParser itself adds a country code if not provided
+  // So if country code is not present in phone_number throw error
+  const app_settings = config.getAll();
+  const validPhone = phoneNumberParser.validate(app_settings, phoneNumber);
```

Comment:
```suggestion
  const appSettings = config.getAll();
  const validPhone = phoneNumberParser.validate(appSettings, phoneNumber);
```

Nitpick: Just a minor style guide change - JS variables should be camel case.